static/js/index.js

```diff
@@ -2508,47 +2487,22 @@ function getQuestIndex (questId) {
     return 2;
 }
 
-function getGymSize (teamId) {
-    const size = gymFilter['t' + teamId].size;
-    if (size === 'huge') {
-        return 75;
-    }
-    return 40;
-}
-
-function getRaidSize (id) {
-    if (raidFilter[id] === undefined) {
-        return 30;
-    }
-    const size = raidFilter[id].size;
-    if (size === 'huge') {
-        return 65;
-    }
-    return 30;
-}
-
-function getPokestopSize (id) {
-    const size = id.includes('i')
-        ? invasionFilter[id].size
-        : pokestopFilter[id].size;
-    if (size === 'huge') {
-        return 65;
-    }
-    return 30;
-}
-
-function getPortalSize (portal, ts) {
-    const yesterday = ts - (60 * 60 * 24);
-    if (portal.imported > yesterday) {
-        // TODO: Portal size
-        if (portalFilter['new'].size === 'huge') {
-            return 50;
-        }
-    }
-    return 25;
+const getIconSize = (type, id, form) => {
+    let filterId
+    let filterType = eval(`${type}Filter`)
+    switch (type) {
+        case 'gym':         filterId = `t${id}`; break;
+        case 'raid':        filterId = form === undefined ? `l${id}` : form === 0 ? `p${id}` : `p${id}-${form}`; break;
+        case 'pokestop':    filterId = id; break;
+        case 'quest':       filterId = id; break;
+        case 'invasion':    filterId = id; break;
+        case 'spawnpoint':  filterId = id; break;
+        case 'pokemon':     filterId = form === 0 ? `${id}-${masterfile.pokemon[id].default_form_id}` : `${id}-${form}`; break;
```

Comment:
How about support for big-karp/small-rat?

Reply:
Added in 9032bd